platforms/Windows/readme.md

```diff
@@ -0,0 +1,305 @@
+# Windows installers
+
+## Online and offline bundles
+
+The Swift SDK bundle is now available in two flavors:
+
+- **Online**: A small .exe that downloads the packages the user selected at install time. Can download fewer total bytes but must be connected to the Internet. The work to stage the payloads required for the online bundle is left as an exercise for the reader.
+- **Offline**: A big .exe that contains all the packages and are extracted on-demand at install time. Downloads the most bytes but doesn't require live Internet connection.
+
+A third pseudo-flavor is using the online bundle with the `/layout` switch to create a layout that can be placed on a network or USB drive to install offline.
+
+### Download URLS
+
+For the online bundle, we need to provide a download URL for each package and its .cabs. (The SourceFile attribute is also required, both for the offline bundle and for Burn to get the hash of the packages. Note that the packages and .cabs available for download must exactly match the ones used when building the bundle. Burn validates downloads by their hashes.)
+
+(For the offline bundle, the download URLs are ignored so we can provide any string. That's easier than conditionalizing the `Chain` of packages and omitting the `DownloadUrl` attributes.)
+
+`BaseReleaseDownloadUrl` is a preprocessor variable that provides the URL to the directory containing the packages for that bundle. So, for example, for the bundle at `https://download.swift.org/swift-5.8.1-release/windows10/swift-5.8.1-RELEASE/swift-5.8.1-RELEASE-windows10.exe`, `BaseReleaseDownloadUrl` would be `https://download.swift.org/swift-5.8.1-release/windows10/swift-5.8.1-RELEASE`.
+
+Burn supports some [special syntax](https://wixtoolset.org/docs/schema/wxs/msipackage/) to simplify authoring download URLs. The one we use is `{2}`, which is replaced by the file name of the payload.
+
+Burn automatically expands payload authoring to handle .cab files that are external to their .msi files so `DownloadUrl="$(BaseReleaseDownloadUrl)/{2}"` turns into, for example:
+
+- `$(BaseReleaseDownloadUrl)/bld.msi`
+- `$(BaseReleaseDownloadUrl)/bld.cab`
+
+
+### Install directory and feature selection
+
+The bundle authoring (in installer.wxs) drives optional install directory and feature selection via `Variable`s that can be specified by the user at the command line and in the bundle UI on the Options page.
+
+| Variable | Description |
+| -------- | ----------- |
+| InstallRoot | A formatted string variable that specifies the installation root directory. The default value specified in installer.wxs should match the equivalent `INSTALLROOT` authoring in shared.wxs. The bundle variable is passed to each `MsiPackage` so overwrites the default directory authored in the MSI packages -- but keeping them in sync avoids the confusion if the default directory should change. |
+| OptionsInstallBld | Controls whether bld.msi will be installed. |
```

Comment:
I wonder if we should revisit this - I'm trying to envision what scenario would the user wish to not install the compiler but install any other component.

Reply:
Easy enough to remove the `InstallCondition`. You could even leave the checkbox and set it to disabled.

Reply:
Oh, I really like that!  (checked and visible but disabled).